exercises/word-count/description.md

```diff
@@ -12,7 +12,7 @@ When counting words you can assume the following rules:
 
 1. The count is _case insensitive_ (ie "You", "you", and "YOU" are 3 uses of the same word)
 2. The count is _unordered_; the tests will ignore how words and counts are ordered
-3. Other than the apostrophe in a _contraction_ all forms of _punctuation_ are ignored
+3. Other than the apostrophe in a _contraction_ all forms of _punctuation_ are regarded as spaces
```

Comment:
What do we do with spaces?  We treat them as part of a word break. It is not identical.  All forms of punction are ignore, and I believe this is correct.

Reply:
What does "ignored" mean? If it means being skipped and left unread then test case _handles cramped lists_ should be

```json
"input": {
    "sentence": "one,two,three"
},
"expected": {
    "onetwothree": 1
}
```

while actually it is

```json
"input": {
    "sentence": "one,two,three"
},
"expected": {
    "one": 1,
    "two": 1,
    "three": 1
}
```

i.e. `,` is treated as a word break. Perhaps a better formulation would be "regarded as work breaks", but the parsing logic is all the same.

Reply:
I am not sure that we can get a canonically accurate word count.  All of the professional word processors get it wrong, or right, even, sometimes.

I was thinking of punctuation as one thing, and non-word things as another.  So punctuation is a word boundary, as is space, vertical tab, while the punctuation that is `-` is not a word boundary, sometimes, and for example.

So my use of "punctuation" in my response was definitely a lazy response.  I guess first we must identify what "all punctuation" means.